chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/RoomSpek.kt

```diff
@@ -170,6 +172,48 @@ class RoomSpek : Spek({
 
             assertThat(badEvents).isEmpty()
         }
+
+        it("always has the correct memberUserIds after room data is changed") {
+
+            setUpInstanceWith(createDefaultRole(), newUsers(PUSHERINO, ALICE), newRoom(GENERAL, ALICE))
+            val superUser = chatFor(SUPER_USER).connect().assumeSuccess()
+            val roomUpdated = CountDownLatch(1)
+
+            superUser.subscribeToRoomMultipart(superUser.generalRoom) { event ->
+                when (event) {
+                    is RoomEvent.RoomUpdated -> {
+                        roomUpdated.countDown()
+                    }
+                }
+            }
+
+            assertThat(superUser.generalRoom.memberUserIds.size).isEqualTo(2) //alice and super user
+            superUser.updateRoom(superUser.generalRoom, customData = mapOf("key" to "data") ).assumeSuccess()
```

Comment:
> "data") ).assumeSuccess()

the autoformatter in Android Studio would catch and correct this stray space and a couple of other inconsistencies in formatting.

Reply:
Done!